build.sh

```diff
@@ -12,31 +12,55 @@ ErrChk
 
 TerraformPath="./terraform/${PROVIDER}"
 AnsFlgs="-i ${TerraformPath}/inventory.yaml"
+#AnsFlgs="${AnsFlgs} -vvvv"
 AnsPlybkPath="./ansible/playbooks"
 
+echo "--QE_SAP_DEPLOYMENT--"
 
 ### TERRAFORM BIT ###
-TF_LOG_PATH=terraform.apply.log TF_LOG=INFO terraform -chdir="${TerraformPath}" apply -auto-approve
+TF_LOG_PATH=terraform.init.log TF_LOG=INFO terraform -chdir="${TerraformPath}" init
```

Comment:
Splitting of PLAN and APPLY is not strictly related to the execution in a container.